graphql/tests/mutation/setPatchVisibility/results.json

```diff
@@ -12,6 +12,25 @@
                     ]
                 }
             }
+        },
```

Comment:
Can we add a graphql test for admins being able to modify this

Reply:
It's already included since the default `test_user_id` is `admin_user` for the first test in this file.